VisualizationBase/src/Scene.cpp

```diff
@@ -139,7 +140,7 @@ void Scene::removeTopLevelItem(Item* item)
 
 ViewItem* Scene::currentViewItem()
 {
-	if (viewItems_.size() == 0)
+	if (viewItems_[0].size() + viewItems_[1].size() + viewItems_[2].size() == 0)
```

Comment:
do not hardcode the fact that there are three columns in code. Use `std::all_of` or something similar: http://www.cplusplus.com/reference/algorithm/all_of/